Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -207,6 +212,66 @@ Describe "New-GcsObject" {
         $obj.Metadata["Content-Type"] | Should Be "image/jpeg"
         Remove-GcsObject $obj
     }
+
+    It "should upload an empty folder" {
+        $TestFolder = Join-Path $TestDrive "TestFolder"
```

Comment:
Thanks for the info, I was unaware of `$TestDrive`. FWIW all of our other Pester tests just read/write files directly from the Temp directory. So I'd actually be in favor of just continuing to do that. However, I don't feel strongly about it. So fee free to keep it in if you like.